aci/convert/convert_test.go

```diff
@@ -724,7 +724,7 @@ func TestConvertSecrets(t *testing.T) {
 	_, err = tmpFile.Write([]byte("test content"))
 	assert.NilError(t, err)
 	t.Cleanup(func() {
-		assert.NilError(t, os.Remove(tmpFile.Name()))
+		_ = os.Remove(tmpFile.Name())
```

Comment:
failure on windows. File should be closed after reading, this is handled OK by File.ReadAll(), but I suppose windows is doing this async (not the first time I see this) and the test cleanup is arriving too quickly and racing with the actual file closing...

Reply:
Or maybe the Windows Defender is grabbing new files? 🤔
I could check if we have disabled it.

Reply:
? maybe (on Github Action windows nodes ? ) I could reproduce this on my windows laptop

Reply:
ok, that was a reflex, I thought it is Jenkins one of the OVH machines 😂